itest/README.md

```diff
@@ -0,0 +1,106 @@
+# Integration Test
+
+`itest` is a package that houses the integration tests made for `lnd`. This
+package builds test cases using the test framework `lntest`.
+
+## Add New Tests
+
+To add a new test case, create a `TestFunc` and add it in `list_on_test.go`.
+Ideally, the `Name` should just be the snake case of the name used in
+`TestFunc` without the leading `test` and underscores. For instance, to test
+`lnd`'s exporting channel backup, we have,
+
+```go
+{
+		Name:     "export channel backup",
+		TestFunc: testExportChannelBackup,
+}
+```
+
+The place to put the code of the `TestFunc` is case-specific. `itest` package
+has loosely defined a list of files to test different functionalities of `lnd`.
+The new test needs to be put into one of these files, otherwise, a new file
+needs to be created.
+
+## Run Tests
+
+#### Run a single test case
+
+To run a single test case, use `make itest icase=$case`, where `case` is the
+name defined in `list_on_test.go`, with spaces replaced with underscores(`_`).
+
+```sh
```

Comment:
nit: we use the more generic `shell` command everywhere else.

Reply:
switched!